packer/files/install_python

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/bash
+print_usage() {
+    echo "install_python [--os centos/ubuntu]"
+    exit 1
+}
+OS="centos"
+while [ $# -gt 0 ]; do
+    case "$1" in
+        "--os")
+            OS="$2"
+            shift 2
+            ;;
+        *)
+            print_usage
+            ;;
+    esac
+done
+if [ "$OS" = "ubuntu" ]; then
+    sudo apt-get install -y python3-setuptools
+    sudo apt-get install -y python3-pip
+    exit 0
+fi
+
+yum -y groupinstall 'Development Tools'
+yum -y install openssl-devel bzip2-devel libffi-devel
+yum remove -y python3
+curl -O https://www.python.org/ftp/python/3.8.10/Python-3.8.10.tgz
```

Comment:
https://www.python.org/downloads/release/python-3818/ perhaps?

Reply:
(I'm somewhat surprised whatever comes with the OS is not good enough, btw)

Reply:
i don't remember which of the ubuntu installation (AWS or gcp) didn't have python installed.